### PR TITLE
Midterm markdown refactor

### PR DESCRIPTION
**Pull request by l-zeuch**
This pull request adds the latest markdown files to `master`, as issue #186 tracks.

It includes a reworked documentation for the afk system as well as the suggestion system.
I think I made a pretty good job at guiding the user through the process of installing, catching possible questions and problems, and explaining the configuration variables, if applicable.

This is more meant as a midterm action, as there are surely still some files to work on, however I think we shouldn't withhold updates for too long.

**Status**
- [x] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [x] I have read and followed the [contribution guide](../CONTRIBUTING.md)

### CODE REVIEW

File: afk/README.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
 # AFK System
+This package includes two custom commands, both of which you will have to add to your server.<br>
+In case you need a refresher, or don't know how to add a custom command, please read [this](https://learn.yagpdb.xyz/the-custom-command-interface) tutorial.
 
-These commands are **not** standalone. Add all the commands if you wish to use them.
-These CCs allow you to create an AFK system for your users.
+## Features
+* Set AFK-status with optional duration
```
Comment (jo3-l) [inline afk/README.md:6]:
```suggestion
* Set AFK status with optional duration
```

File: afk/afk.go.tmpl [hunk 2]
```diff
@@ -5,11 +5,13 @@
 	-afk <message>
 	-afk <message> -d <duration>
 
-	Recommended trigger: Regex trigger with trigger `.*`
+	Recommended trigger: Regex trigger with trigger `\A`
+
+    Author: DaviiD1337 <https://github.com/DaviiD1337>
```
Comment (jo3-l) [inline afk/afk.go.tmpl:10]:
```suggestion
	Recommended trigger: Regex trigger with trigger `\A`

	Author: DaviiD1337 <https://github.com/DaviiD1337>
```
Tabs over spaces.

File: afk/afk.go.tmpl [hunk 3]
```diff
@@ -5,11 +5,13 @@
...
 */}}
 
 {{/* CONFIGURATION VALUE STARTS HERE */}}
-{{ $removeAfkOnMessage := true}}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
+{{ $removeAfkOnMessage := true}}{{/* Remove the AFK status from an AFK user when they send a message. */}}
```
Comment (jo3-l) [inline afk/afk.go.tmpl:14]:
```suggestion
{{ $removeAfkOnMessage := true}}{{/* Whether or not to remove the AFK status from users when they send a message. */}}
```

File: suggestion/README.md [hunk 4]
```diff
@@ -1,117 +1,138 @@
...
-| --- |
+###### Suggestion System Trigger
+```
+(?i)\A(\-\s?|<@!?204255221017214977>\s*)((del(ete)?|edit)?suggest(ion)?|(sa|suggestadmin)\s+((?:mark)?dupe|deny|implement(ed)?|archive|approved?|comment))(\s+|\z)
+```
 
+| ⚠ Make sure to replace the `-` at the beginning of the trigger with your server's prefix.
+| ---- |
+
+You will not be able to save yet, we have to do one minor tweak: Remove the large leading comment as well as the second comment:
```
Comment (jo3-l) [inline suggestion/README.md:25]:
```suggestion
You will not be able to save yet, as the custom command code is slightly over the limit as is. To fix this, we have to do one minor tweak: remove the large leading comment as well as the second comment:
```

Brief explanation as to why it does not save.

File: suggestion/README.md [hunk 5]
```diff
@@ -1,117 +1,138 @@
...
+| ---- |
+
+* `$Suggestion_Channel`<br>
+   This is the main channel, where all suggestions made through the command will show up - This is the `#suggestions` channel on the support server.
+* `$Logging_Channel`<br>
+   This is the channel where the authors are being notified if a suggestion was denied, approved, implemented, or marked as dupe. In the YAGPDB server, this is the `#suggestion-discussion` channel.
+* `$Implemented_Channel`<br>
+   The channel where suggestions which have been marked as implemented are sent. This provides a good way to organise all implemented suggestions into a separate channel. On the support server, you can find this channel as `#implemented-suggestions`.
+* `$Approved_Channel`
+   Where approved suggestions are being sent to. This is not quite equal to implemented suggestions: You can see approved suggestions as "being worked on", and implemented suggestions as "this is now a feature". In the Support Server, we log those under `#implemented-suggestions` as well.
```
Comment (jo3-l) [inline suggestion/README.md:58]:
```suggestion
   Where approved suggestions are being sent to. This is not the same as implemented suggestions: you can see approved suggestions as "being worked on", and implemented suggestions as "this is now a feature". In the Support Server, we log those under `#implemented-suggestions` as well.
```
Reads slightly better.

File: suggestion/README.md [hunk 6]
```diff
@@ -1,117 +1,138 @@
...
+Okay, there are still a few things to do before your suggestion system is set up, which are the configuration variables. Please take your time to read through their description before making any changes.
+
+### Channels 
+| ℹ Among these variables, all channels can be separate or the same. |
+| ---- |
+
+* `$Suggestion_Channel`<br>
+   This is the main channel, where all suggestions made through the command will show up - This is the `#suggestions` channel on the support server.
+* `$Logging_Channel`<br>
+   This is the channel where the authors are being notified if a suggestion was denied, approved, implemented, or marked as dupe. In the YAGPDB server, this is the `#suggestion-discussion` channel.
```
Comment (jo3-l) [inline suggestion/README.md:54]:
```suggestion
   This is the channel where the authors are notified if a suggestion was denied, approved, implemented, or marked as dupe. In the YAGPDB server, this is the `#suggestion-discussion` channel.
```

File: suggestion/README.md [hunk 7]
```diff
@@ -1,117 +1,138 @@
...
+### For Mods / Admins
+| ℹ All these commands are to be preceded with `sa` or `suggestadmin`. |
+| ---- |
+
+* `deny`<br>
+   Syntax: `deny <Suggestion-ID:Whole Number> [Reason:Text]`<br>
+   Use: Deny a suggestion and notify the author that their suggestion has been deleted, along with the optional reason for denial.
+* `dupe`<br>
+   Syntax: `dupe <Suggestion-ID:Whole Number> <Old Suggestion-ID:Whole Number>`<br>
+   Use: Mark the first given suggestion as dupe of the second suggestion, delete the duplicate and inform the author of the duplicate.
```
Comment (jo3-l) [inline suggestion/README.md:116]:
```suggestion
   Use: Mark the first given suggestion as a dupe of the second suggestion, delete the duplicate and inform the author of the duplicate.
```

File: suggestion/README.md [hunk 8]
```diff
@@ -1,117 +1,138 @@
...
+* `approve`<br>
+   Syntax: `approve <Suggestion-ID:Whole Number> [Comment:Text]`<br>
+   Use: Approve of a suggestion and log it to the channel for approved suggestions and notify the author. Adds a record of who approved the suggestion.
```
Comment (jo3-l) [inline suggestion/README.md:119]:
```suggestion
   Use: Approve a suggestion and log it to the channel for approved suggestions and notify the author. Adds a record of who approved the suggestion.
```

**Review by jo3-l — CHANGES_REQUESTED** [review]
Some minor wordsmithing. Content generally looks fine.

**Comment by l-zeuch** [comment]
Thanks for the feedback!
I'll quickly implement the suggested changes!

**Comment by l-zeuch** [comment]
Hold on a minute, somehow git did not track all changes. I'll push again.

**Comment by l-zeuch** [comment]
everything should be there now, if not, I've got time 😄

**Review by jo3-l — APPROVED** [review]

**Review by DaviiD1337 — APPROVED** [review]

**Comment by l-zeuch** [comment]
Thanks for the feedback you two, I'll merge it now!

**Pull request merged**